entities/SpaceShip.cs

```diff
@@ -17,7 +16,7 @@ public SpaceShip(string name)
 
 		public SpaceShip(string name, params int[] payloadAndSystems)
```

Comment:
Możliwe użycie ICollection

Reply:
Muszę użyć *varargs* (tu: *params*), za to są punkty.